src/test/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/CaseAccessMetadataTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.GrantType.BASIC;
+import static uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.GrantType.CHALLENGED;
+import static uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.GrantType.EXCLUDED;
+import static uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.GrantType.STANDARD;
+
+class CaseAccessMetadataTest {
+
+    private CaseAccessMetadata caseAccessMetadata = new CaseAccessMetadata();
+
+    @Test
+    void testGetAccessGrantsString() {
+        caseAccessMetadata.setAccessGrants(List.of(CHALLENGED, BASIC, STANDARD, EXCLUDED));
+
+        String expectedSortedValue = BASIC + "," + CHALLENGED + "," + EXCLUDED + "," + STANDARD;
```

Comment:
Could add a comma delimiter final field if used frequently here to join these up

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/ccd-data-store-api/pull/1479/commits/528c646b730946ed64922e15dd898b8b992e5f6d